ui/src/app/components/notification-banner.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+
+import {FlexRow} from "./flex";
+import {AlarmExclamation} from "./icons";
+import {Button} from "./buttons";
+import {reactStyles} from 'app/utils';
+import colors from 'app/styles/colors';
+
+const styles = reactStyles({
+    box: {
+        boxSizing: 'border-box',
+        height: '56.5px',
+        width: '380.5px',
```

Comment:
I thought the IAL2 and AAR notification have different width is because the text length are different. With the command length, I am worried some future notification has longer text and would become a 3-row banner.

Reply:
Shomin + Lou prefer this version.

If we need to adjust the width in the future for longer messages, that's easy to do.

Reply:
okay. Is that easy to make this customizable like IAL2 is wider than the other? But we can worry about that in the future